### ISSUE TITLE
Docs tables could use more contrast

### ISSUE THREAD

**Issue by dtuite** [backstage/backstage#5264]
## Expected Behavior

It should be possible to discern the borders of markdown tables which are rendered in the docs. For example, see GitHub in dark mode:

<img width="588" alt="Screenshot 2021-04-08 at 21 43 54" src="https://user-images.githubusercontent.com/562403/114093921-974e6f80-98b3-11eb-970d-56079d3f026f.png">

It should also be possible to discern the titles and know that they are distinct to the body.

## Current Behavior

Borders are barely discernable in light mode and invisible in dark mode.

<img width="633" alt="Screenshot 2021-04-08 at 21 41 29" src="https://user-images.githubusercontent.com/562403/114093987-acc39980-98b3-11eb-936e-6aba0ad9994e.png">

<img width="584" alt="Screenshot 2021-04-08 at 21 42 00" src="https://user-images.githubusercontent.com/562403/114093999-b0efb700-98b3-11eb-8526-0fb90145a8f6.png">

## Steps to Reproduce

Add [this service](https://github.com/RoadieHQ/sample-service/blob/main/catalog-info-1.yaml) to your catalog, navigate to the docs for this service and open the "Feature testing" doc.

## Your Environment

Chrome on MacOS
